debian/changelog

```diff
@@ -1,3 +1,9 @@
+kano-desktop (3.11.1-0) unstable; urgency=low
+
+  * Added a Dashboard bootup welcome sound
```

Comment:
Let's add this feature to the log below `kano-desktop (3.11.0-0) unstable; urgency=low` such that we have all the features we added for this package match the OS version.